Gem/Code/Source/Components/NetworkStressTestComponent.cpp

```diff
@@ -38,7 +38,9 @@ namespace MultiplayerSample
 
     NetworkStressTestComponentController::NetworkStressTestComponentController(NetworkStressTestComponent& owner)
         : NetworkStressTestComponentControllerBase(owner)
+#if AZ_TRAIT_SERVER_ENABLED
         , m_autoSpawnTimer([this]() { HandleSpawnAiEntity(); }, AZ::Name("StressTestSpawner Event"))
+#endif
```

Comment:
This pattern is something I'm not particularly fond of. It occurs because HandleSpawnAiEntity is not declared on the Client. An alternative is to declare it on the client but define it as empty. The tradeoff here is that the Client has a concept of HandleSpawnAiEntity in exchange for simplifying some of the macro usage around it.